tooling/cli/src/helpers/template.rs

```diff
@@ -80,13 +80,14 @@ pub fn render_with_generator<
         file_path.set_extension("toml");
       }
     }
-    let mut output_file = out_file_generator(&file_path)?;
```

Comment:
cargo-mobile handles this with the Filter enum instead of ignoring errors here, I think that would be a better fit

for instance this would ignore permission errors and the user might not know what's wrong in their template

Reply:
do you want the same mechanism? as in read the gitignore file and depending on that, overwrite? because that would mean it won't override anyfiles

Reply:
I could change the `out_file_generator` to return `Result<Option<File>>` and skip if `None`

Reply:
that works for now. I think parsing gitignore is a bit too much